src/components/App/App.test.js

```diff
@@ -7,12 +7,21 @@ describe('App', () => {
 
         const headerComponent = screen.queryByTestId("header");
         expect(headerComponent).toBeInTheDocument();
-      });
-      
-      test('should render a view', () => {
-          render(<App />);
+    });
 
-          const view = screen.queryByTestId("view");
-          expect(view).toBeInTheDocument();
-      });
+    //TEST NOT WORKING WITH HEADER BY ROLE, TODO: FIND OUT WHY
+
+    // test('should render the header', () => {
+    //     render(<App />);
+
+    //     const headerComponent = screen.queryByRole("heading");
+    //     expect(headerComponent).toBeInTheDocument();
+    // });
```

Comment:
recuerda que puedes usar la función `test.todo` para casos como este
https://dev.to/karltaylor/use-test-todo-when-writing-jest-tests-4dbd

Reply:
Revisaré cómo funciona, gracias!

Reply:
Interesante, es bueno saber que existe esta función. La utilizo en el commit bc55c63 pero me temo que aquí voy a necesitar tu ayuda, no he descubierto qué estoy haciendo mal o por qué me está dando el error `TestingLibraryElementError: Found multiple elements with the role "heading"` siendo que solamente existe un elemento con `role="heading"`. Alguna idea de qué puede estar pasando?

Reply:
Lo elementos que tienen `role=heading` son `<h1>-<h6>`, que creo que tienes varias en la app, entonces me hace sentido que te arroje error. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
Ahora, si lo que quieres es obtener el `<header>`, ese tiene un `role=banner`. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_role
Un poco confuso, no?

Reply:
Tienes toda la razón! Siendo sincero no sabía que algunos elementos un `role` implícito. De nuevo te agradezco por iluminarme 🤝

Reply:
Corregido en baf0415